### DISCUSSION TITLE
Quadratic rewards for extra flips

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-docs#112] [IIPs]
Change in flip rewards distribution. Calculate rewards for extra flips in proportion to the size of the stake to the power of `0.9`.
Read more details [here](https://docs.idena.io/docs/iip/iip-7).

**Comment by TravCrypto** [comment]
I agree with IIP-7.

**Comment by dsilberschmidt** [comment]
I may agree, but in my opinion this is not the main problem. I always made 5 flips till one of mine was reported and lost all the reward. Was a surprise after 19 phases. Fix first the reporting problem encourage by the platform. Arbitrary reports makes very risky any aditional flip. And the cocient reportable flips/total flips was arbitrary increased either making the probability of having a flip reported increased substantially. In alternative, or in addition, i would suggest that the lost of rewards happens if you have less that "3 not reported flips". In that way aditional flips over 3 would be an insurance to the lost of rewards for a report.

**Reply by midenaio** [reply]
To prevent false reports >50% of the votes are required for reporting a flip. For smaller committees even more stringent consensus requirements apply.

To investigate your case please send us a link to the reported flip and a screenshot of the flip to `info@idena.io`
You can take a screenshot of the flip in the app Flips>Archive

**Reply by dsilberschmidt** [reply]
That happened on epoch #92 and #94. The flips are not available anymore in the app. Anyway if the app is encouraging to use all five reports from a total of seventeen...assuming all flips of equal quality...any additional flip would have a 5/17 chance of being reported by each member of the committee.  That leads to a lot of valid flips being reported. 
For example in epoch #94 from 13909 flips 1767 were reported, ~ 13%. Hypothesis, many of those 1767 flips were valid but were  over reported.  A simple initial solution would be to discourage somehow reports of valid flips and stop encouraging to use all the reports available.

**Reply by dsilberschmidt** [reply]
¿Do you agree? ¿Do you disagree? I am sending reported flips to info@idena.io. If people is encouraged to use 8 reports from 24 flips is normal that good flips will get reported!
